tensorflow/lite/delegates/vx-delegate/CMakeLists.txt

```diff
@@ -25,13 +25,10 @@ list(APPEND TFLITE_DELEGATES_VX_SRCS
   ${TF_SOURCE_DIR}/core/platform/default/env_time.cc
 )
 
-list(APPEND TFLITE_DELEGATES_VX_DEPENDENCIES
-  "-ltim-vx"
-  "-lCLC"
-  "-lGAL"
-  "-lOpenVX"
-  "-lOpenVXU"
-  "-lVSC"
-  "-lArchModelSw"
-  "-lNNArchPerf")
+if(NOT TIM_VX_INSTALL OR NOT EXISTS ${TIM_VX_INSTALL})
+  message(FATAL_ERROR "Set TIM_VX_INSTALL to tim-vx install folder(by make install) instead of ${TIM_VX_INSTALL}")
+endif()
 
+message(${TIM_VX_INSTALL})
+include_directories(${TIM_VX_INSTALL}/include)
+list(APPEND TFLITE_DELEGATES_VX_DEPENDENCIES ${TIM_VX_INSTALL}/lib/libtim-vx.so)
```

Comment:
The GPU driver libraries are not needed (CLC, GAL, OpenVX, OpenVXU, VSC, ArchModelSw, NNArchPerf) ?